modules/service-fips.sh

```diff
@@ -83,6 +150,21 @@ fips_check_support() {
     esac
 }
 
+fips_print_status() {
+    local result=0
+
+    _fips_updates_is_enabled || result=$?
+    if [ $result -eq 0 ]; then
+        echo "fips-updates (uncertified): enabled"
+    else
+        echo "fips-updates (uncertified): disabled"
+    fi
```

Comment:
Hm, with this change, the output isn't valid yaml anymore. For example:
```
ubuntu@xenial-ua:~$ ua status
esm: disabled (not available)
fips: enabled
  fips-updates (uncertified): enabled
livepatch: enabled
  client-version: "7.23"
  architecture: x86_64
  cpu-model: Intel Core Processor (Skylake)
  last-check: 2018-03-20T19:23:21.463449773Z
  boot-time: 2018-03-20T19:18:34Z
  uptime: 4m51s
  status:
  - kernel: 4.4.0-1003.3-fips
    running: true
    livepatch:
      checkState: checked
      patchState: nothing-to-apply
      version: ""
      fixes: ""
```
If I feed that to python3-yaml, I get:
```
yaml.scanner.ScannerError: mapping values are not allowed here
  in "<unicode string>", line 3, column 29:
      fips-updates (uncertified): enabled
                                ^
```

Any suggestions?

Reply:
Hmm, scratch that, it's already a problem with the others.

Reply:
OK